Description: qt5-fix
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 deepin-screen-recorder (5.10.2-1ubuntu1) impish; urgency=medium
 .
   * New upstream release for impish.
Author: Ananta Mishra <mishraananta2017@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2021-10-14

--- deepin-screen-recorder-5.10.2.orig/src/event_monitor.cpp
+++ deepin-screen-recorder-5.10.2/src/event_monitor.cpp
@@ -28,6 +28,7 @@
 #include <X11/keysymdef.h>
 #include <X11/keysym.h>
 #include <X11/extensions/XTest.h>
+#undef min
 
 #include <iostream>
 #include <dlfcn.h>
--- deepin-screen-recorder-5.10.2.orig/src/screen_shot_event.cpp
+++ deepin-screen-recorder-5.10.2/src/screen_shot_event.cpp
@@ -19,6 +19,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <iostream>
 #include "screen_shot_event.h"
 #include "keydefine.h"
 
@@ -29,8 +30,6 @@
 #include <X11/keysym.h>
 #include <X11/extensions/XTest.h>
 
-#include <iostream>
-
 ScreenShotEvent::ScreenShotEvent(QObject *parent) : QThread(parent)
 {
 
